src/View/Helper/UserHelper.php

```diff
@@ -167,10 +169,29 @@ public function addReCaptcha()
             );
         }
         $this->addReCaptchaScript();
-        try {
-            $this->Form->unlockField('g-recaptcha-response');
-        } catch (\Exception $e) {
+        $version = Configure::read('Users.reCaptcha.version', 2);
+        $method = "addReCaptchaV$version";
+        if (method_exists($this, $method)) {
+            try {
+                $this->Form->unlockField('g-recaptcha-response');
+            } catch (Exception $e) {
```

Comment:
could we check for existing Form helper instead of silent capture the exception here?